backend-flask/README.txt

```diff
@@ -0,0 +1,70 @@
+INSTALL REQUIREMENTS
+You can install flask and other requirements such as RESTfulapi for flask by simply running this script:
+
+$ pip install -r requirements.txt
+
+-------------------------
+VENV
+To install venv enter the following command:
+
+$ python3 -m venv venv
+
+This installs the venv package and names the folder "venv".
+
+To activate the VM, navigate to the directory where your venv folder is located.
+
+For linux users:
+$ source venv/bin/activate
+
+For windows users using cmd:
+> venv\Scripts\activate
+
+Powershell script:
+> venv\Scripts\activate.ps1
+
+Note*: If Powershell gives an execution error, just use cmd or follow this link to see execution policy documentation: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-5.1
+
+To exit out of the VM:
+$ deactivate
+
+-------------------------
+RUNNING THE APP
+In order to run the app, first you have to navigate to the directory the app is stored in. Than you must set an environment variable to the file that we want to be our flask application.
+For example, if our application was named DND.py we would input the following flag in the terminal:
+
+For Linux:
+$ export FLASK_APP=DND.py
+
+For Windows:
+>set FLASK_APP=DND.py
+
+To actually run the application, input the following:
+$ flask run
+    or
+$ python DND.py
```

Comment:
From the docs: 
"from Flask 0.11 onward the flask command is recommended. The reason for this is that due to how the dev server's reload mechanism works there are some bizarre side-effects when using app.run (like executing certain code twice, sometimes crashing without message or dying when a syntax or import error happens)."

If "flask run" is the only command that works its fine because its what they recommend for the above reasons.